pkg/rpc/server/server.go

```diff
@@ -301,7 +301,7 @@ func (s *Server) handleHTTPRequest(w http.ResponseWriter, httpRequest *http.Requ
 			s.writeHTTPErrorResponse(
 				request.NewIn(),
 				w,
-				response.NewInternalServerError("websocket users limit reached", fmt.Errorf("%d subscribers are allowed at max", maxSubscribers)),
+				response.NewInternalServerError(fmt.Errorf("websocket users limit reached: %d subscribers are allowed at max", maxSubscribers)),
```

Comment:
We're not supposed to tell `%d` to the user, and server admin knows the number anyway.